api/nakadi-event-bus-api.yaml

```diff
@@ -797,7 +797,74 @@ paths:
           description: Service (temporarily) unavailable
           schema:
             $ref: '#/definitions/Problem'
+    get:
+      tags:
+        - managed-api
+      security:
+        - oauth2: ['nakadi.event_stream.read']
+      description: Lists all subscriptions that exist in a system.
```

Comment:
I remember we wanted to have it available only for "admin" scope.
The only problem I have with that - if someone will loose the id of subscription he created, it will not be possible for him to find it in any way. So maybe it will make sense to make this list-all endpoint to be available to everyone.